src/transformers/generation/logits_process.py

```diff
@@ -438,6 +438,22 @@ def __call__(self, input_ids: torch.LongTensor, scores: torch.FloatTensor) -> to
 
 
 def _get_ngrams(ngram_size: int, prev_input_ids: torch.Tensor, num_hypos: int):
+    """
+    Parameters:
+    The key is a generated word and the value is a list of the next words observed.
+        ngram_size (`int`):
+            `ngram_size` that can only occur once.
+        prev_input_ids (`torch.Tensor`):
+            A tensor containing tokenized input for each hypothesis.
+        num_hypos (`int`):
+            The number of hypotheses for which n-grams need to be generated.
+
+    Returns:
+        Dictionary of generated ngrams. With an example of no_repeat_ngram =2, assuming input tokenized tensor([[ 40,
+        2883, 2712, 4346, 319, 262, 21511]]) generated ngrams look like this {(40,): [2883, 1053], (2883,): [2712],
+        (2712,): [4346]}.
```

Comment:
I would either:
a) remove the example
b) move the example to before `Args`, in a sentence explaining what the function does and showing an example

Reply:
Let me try again :)